### PR TITLE
Updates the CI config to fix nightly jobs

### PR DESCRIPTION
**Pull request by dkliban**
[noissue]

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -124,29 +160,39 @@ jobs:
...
-        if: failure()
+      - name: Publish docs to pulpproject.org
         run: |
-          http --timeout 30 --check-status --pretty format --print hb http://pulp/pulp/api/v3/status/ || true
-          docker images || true
-          docker ps -a || true
-          docker logs pulp || true
-          docker exec pulp ls -latr /etc/yum.repos.d/ || true
-          docker exec pulp cat /etc/yum.repos.d/* || true
-          docker exec pulp pip3 list
```
Comment (gerrod3) [inline .github/workflows/nightly.yml:152]: Why was the after failure removed? No longer needed?

File: .github/workflows/scripts/publish_plugin_pypi.sh [hunk 2]
```diff
@@ -23,9 +23,9 @@ fi
 
 pip install twine
 
-twine check dist/pulp-ansible-$1.tar.gz || exit 1
 twine check dist/pulp_ansible-$1-py3-none-any.whl || exit 1
-twine upload dist/pulp-ansible-$1.tar.gz -u pulp -p $PYPI_PASSWORD
+twine check dist/pulp-ansible-$1.tar.gz || exit 1
 twine upload dist/pulp_ansible-$1-py3-none-any.whl -u pulp -p $PYPI_PASSWORD
+twine upload dist/pulp-ansible-$1.tar.gz -u pulp -p $PYPI_PASSWORD
```
Comment (gerrod3) [inline .github/workflows/scripts/publish_plugin_pypi.sh:29]: :+1:

**Review by gerrod3 — APPROVED** [review]
@dkliban Looks good, was wondering if I missed something on the changes to `nightly.yml`.

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (dkliban) [reply .github/workflows/nightly.yml:152]: The publish step no longer starts up a container. It only publishes the things that were already generated in previous steps.

Comment (gerrod3) [reply .github/workflows/nightly.yml:152]: Ok cool, I'll merge it.

**Pull request merged**